api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -7,17 +7,14 @@
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified">
     <addColumn tableName="user">
       <column name="creation_time" type="datetime">
-        <constraints nullable="false" />
```

Comment:
[Prod safety comment] How sure are we that it's safe to modify an existing Liquibase change in-place? Will we have to do something manually to make sure the new version of this change gets applied in our test environment? What happens if someone tries to rollback this PR and/or https://github.com/all-of-us/workbench/pull/2948 on their own? (I suspect something will likely go wrong in our prod DB.) We should think of a clear way to signal to the oncall next week that this PR isn't atomically rollback-safe.

Reply:
I think the issue is that something may go wrong if we perform a rollback to this PR or the previous one since the liquibase commits are split between two files.

I think we might actually be OK given that we don't execute any database rollbacks ATM so it should just be the Java/UI code that switches back.

This made me think of an interesting point though. We may need to mark some commits on master as invalid candidates for rollback because it contains a known bug. Previously, I was only thinking about incompatibilities due to database changes.